.github/workflows/create-docker-images.yml

```
@@ -0,0 +1,30 @@
+# Creates a new docker image and pushes it to DockerHub when a version tag is created.
+name: create-docker-images
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  # Based on instructions from https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
+  build-frontend-and-backend:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to docker hub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: Install Docker Compose
+        run: |
+          sudo curl -L "https://github.com/docker/compose/releases/download/$(curl -s https://api.github.com/repos/docker/compose/releases/latest | grep tag_name | cut -d '"' -f 4)/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+          docker-compose --version
+      - name: Build and push images
+        run: |
+          docker-compose -f docker-compose.yml build
+          docker-compose -f docker-compose.yml push
```

Docker compose should already be available, it's just that you don't need the hyphen 😅 See https://github.com/orgs/community/discussions/116610 for more details.
```suggestion
          docker compose -f docker-compose.yml build
          docker compose -f docker-compose.yml push
```